wc-smooth-generator.php

```diff
@@ -57,3 +57,20 @@ function load_wc_smooth_generator() {
 		\Automattic\WooCommerce\Utilities\FeaturesUtil::declare_compatibility( 'custom_order_tables', __FILE__, true );
 	}
 } );
+
+/**
+ * Show action links on the plugin screen.
+ *
+ * @param mixed $links Plugin Action links.
+ *
+ * @return array
+ */
+function wc_smooth_generator_plugin_action_links( $links ) {
+	$action_links = array(
+		'settings' => '<a href="' . admin_url( 'tools.php?page=smoothgenerator' ) . '" aria-label="View WooCommerce Smooth Generator settings">Settings</a>',
```

Comment:
```suggestion
		'settings' => '<a href="' . esc_url( admin_url( 'tools.php?page=smoothgenerator' ) ) . '" aria-label="' . esc_attr__( 'View WooCommerce Smooth Generator settings', 'wc-smooth-generator' ) . '">' . esc_html__( 'Settings', 'wc-smooth-generator' ) . '</a>'
```
URL escaping and i18n